som/display.go

```diff
@@ -118,18 +118,30 @@ func UMatrixSVG(codebook *mat64.Dense, dims []int, uShape, title string, writer
 		polygonCoords := ""
 		x := scale(coord.At(0, 0))
 		y := scale(coord.At(1, 0))
-		xOffset := 0.5 * MUL
-		yOffset := 0.5 * MUL
 		// hexagon has a different yOffset
 		if uShape == "hexagon" {
```

Comment:
This would be nicer with `switch` and also more `Go`-ish. Go hates `else` clauses 😇